SpywareFilter/sections/specific.txt

```diff
@@ -47,6 +47,7 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|tozlu.com|elleshoes.com|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|partifabrik.com|newsmax.com
 !
+||istiklal.com.tr/hit-increment
```

Comment:
`.com/hit-increment/` > `/hit-increment/`

Reply:
Added it directly, but if this rule break something on other sites, please change it to the specific one -> `||istiklal.com.tr/hit-increment/`.